src/benchmark.rs

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2019 Gabriela Alexandra Moldovan
-// Copyright (c) 2019 King's College London created by the Software Development Team
+// Copyright (c) 2019 King's College London (The Software Development Team)
```

Comment:
Sorry to be a licence pedant, but unfortunately we have to say "created by ..." :(

If @vext01 is OK with it, how about "Copyright (C) 2019 King's College London.\nCreated by the Software Development Team <https://soft-dev.org/>"? I know it's a mouthful :(

Reply:
Ah, OK. Sorry about that Gabi. My bad!

Reply:
No problem!